src/main/scala/ScalatraBootstrap.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Part of NDLA concept_api.
```

Comment:
Bindestrek og ikkje underscore.
```suggestion
 * Part of NDLA concept-api.
```